diff --git a/Makefile b/Makefile
index 00bc5c2..eabfc67 100644
--- a/Makefile
+++ b/Makefile
@@ -149,14 +149,14 @@ proto/felixbackend.pb.go: proto/felixbackend.proto
 		      $(PROTOC_CONTAINER) \
 		      $(PROTOC_IMPORTS) \
 		      proto/*.proto \
-		      --gogofast_out=plugins=grpc:proto
+		      --gogofaster_out=plugins=grpc,$(PROTOC_MAPPINGS):proto
 
 proto/healthz.pb.go: proto/healthz.proto
 	$(DOCKER_RUN) -v $(CURDIR):/src:rw \
 		      $(PROTOC_CONTAINER) \
 		      $(PROTOC_IMPORTS) \
 		      proto/*.proto \
-		      --gogofast_out=plugins=grpc:proto
+		      --gogofaster_out=plugins=grpc,$(PROTOC_MAPPINGS):proto
 
 
 # Building the image
